src/Pos/DB/Block.hs

```diff
@@ -225,28 +225,25 @@ delete :: (MonadDB m) => ByteString -> m ()
 delete k = rocksDelete k =<< getBlockIndexDB
 
 getData ::  (MonadIO m, MonadCatch m, Bi v) => FilePath -> m (Maybe v)
-getData fp = liftIO (decodeMaybe . BSL.fromStrict <$> BS.readFile fp) `catch` handle
-  where
-    decodeMaybe x = case decodeFull x of
-        Left _  -> Nothing
-        Right r -> Just r
-    handle (_::SomeException) = pure Nothing
+getData fp = liftIO (rightToMaybe . decodeFull . BSL.fromStrict <$> BS.readFile fp)
```

Comment:
No, I want to see an exception if decoding fails. That's what we do when we are working with rocksdb. Also I suspect that exception will be throw if file doesn't exist. But in this case I find it more reasonable to return `Nothing`.